dz15/src/app.js

```diff
@@ -0,0 +1,127 @@
+async function getInfoFromDummyjsonProductsCategories() {
+  const response = await fetch('https://dummyjson.com/products/categories')
+  const data = response.json()
+  return data
+}
+
+async function getCountFromDummyjsonProductsCategories() {
+  const counter = await getInfoFromDummyjsonProductsCategories()
+  return counter.length
+}
+
+async function findCategories(data, categories) {
```

Comment:
```suggestion
function findCategories(data, categories) {
```

эта функция ничего асинхронного не делает, просто ищет по массиву - можно убирать async